docs/cli-tool.md

```diff
@@ -0,0 +1,38 @@
+# CLI Tool
+
+[jvsctl](../cmd/jvsctl) facilitates the justification verification flow provided
+by abcxyz/jvs
+
+## Usage
+
+jvsctl [command]
+
+### Global Flags:
```

Comment:
Add a section for config file https://github.com/abcxyz/jvs/blob/main/pkg/config/cli_config.go

Reply:
done